pyproject.toml

```diff
@@ -38,7 +38,7 @@ numpy = ">=1.22"
 grpcio-health-checking = ">=1.43"
 packaging = ">=15.0"
 typing-extensions = ">=4.5.0"
-ansys-api-acp = "^0.1.dev9"
+ansys-api-acp = ">=0.2.0.dev0,<0.4.0"
```

Comment:
Why 0.4 and not 0.3?

Reply:
Should have been 0.3, the change was unintentional.